modules/ROOT/pages/platform/metrics-integration.adoc

```diff
@@ -0,0 +1,430 @@
+| `neo4j_dbms_bolt_connections_closed_total`
+| _Counter_
+| MAX
+
+| Bolt Connections Opened
+| The total number of Bolt connections opened since startup. This includes both successful and failed connections. This value may drop if background maintenance is performed by Aura.
+| `neo4j_dbms_bolt_connections_opened_total`
+| _Counter_
+| MAX
+
+| Garbage Collection Young Generation
+| Shows the total time since startup spent clearing up heap space for short lived objects. Young garbage collections typically complete quickly, and the Aura instance will wait whilst the garbage collector is run. High values indicate that the instance is running low on memory for the workload and you should consider increasing the size of your instance.
+| `neo4j_dbms_vm_gc_time_g1_young_generation_total`
+| _Counter_
+| MAX
+
+| Garbage Collection Old Generation
+| Shows the total time since startup spent clearing up heap space for long lived objects. Old garbage collections can take time to complete, and the Aura instance will wait whilst the garbage collector is run. High values indicate that there are long running processes or queries which could be optimized, or your instance is running low on CPU or memory for the workload and you should consider reviewing these metrics and possibly increasing the size of your instance.
+| `neo4j_dbms_vm_gc_time_g1_old_generation_total`
+| _Counter_
+| MAX
+
+| Replan Events
+| The total number of times Cypher has replanned a query since the server started. If this spikes or is increasing, check that the queries executed are using parameters correctly. This value may drop if background maintenance is performed by Aura.
+| `neo4j_database_cypher_replan_events_total`
+| _Counter_
+| MAX
+
+| Active Read Transactions
+| The number of currently active read transactions.
+| `neo4j_database_transaction_active_read`
+| _Gauge_
+| MAX
+
+| Active Write Transactions
+| The number of currently active write transactions.
+| `neo4j_database_transaction_active_write`
+| _Gauge_
+| MAX
+
+| Committed Transactions
+| The total number of committed transactions since the server was started. This value may drop if background maintenance is performed by Aura.
+| `neo4j_database_transaction_committed_total`
+| _Counter_
+| MAX
+
+| Peak Concurrent Transactions
+| The highest number of concurrent transactions detected since the server started. This value may drop if background maintenance is performed by Aura.
+| `neo4j_database_transaction_peak_concurrent_total`
+| _Counter_
+| MAX
+
+| Transaction Rollbacks
+| The total number of rolled back transactions. This value may drop if background maintenance is performed by Aura.
+| `neo4j_database_transaction_rollbacks_total`
+| _Counter_
+| MAX
+
+| Checkpoint Events
+| The total number of checkpoint events executed since the server started. This value may drop if background maintenance is performed by Aura.
```

Comment:
```suggestion
| The total number of checkpoint events executed since the server started. 
This value may drop if background maintenance is performed by Aura.
```